esmvalcore/_provenance.py

```diff
@@ -31,9 +32,12 @@ def create_namespace(provenance, namespace):
 def get_esmvaltool_provenance():
     """Create an esmvaltool run activity."""
     provenance = ProvDocument()
-    namespace = 'software'
-    create_namespace(provenance, namespace)
-    attributes = {}  # TODO: add dependencies with versions here
+    for namespace in ('software', 'attribute'):
+        create_namespace(provenance, namespace)
+
+    # TODO: add dependencies with versions here
+    attributes_value = ESMVALTOOL_PAPER_TAG
+    attributes = {'attribute:references': attributes_value}
```

Comment:
```suggestion
    attributes = {'attribute:references': ESMVALTOOL_PAPER_TAG}
```
would be a bit shorter